Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -715,7 +715,12 @@ class RenderStyle {
 
     inline ContentVisibility contentVisibility() const;
 
-    inline std::optional<ContentVisibility> skippedContentReason() const;
+    // effectiveContentVisibility will return ContentVisibility::Hidden if a content-visibility: hidden
+    // subtree (overriding content-visibility: auto at all times), ContentVisibility::Auto in a
+    // content-visibility: auto subtree, and ContentVisibility::Visible otherwise.
```

Comment:
I think it's important to clarify that in the `auto` case, we only return `auto` when the content is not relevant (or skipped).

Reply:
Good point, fixed.